src/main/java/me/rigamortis/seppuku/impl/module/render/WallHackModule.java

```diff
@@ -58,36 +44,36 @@
     public final BooleanValue mobs = new BooleanValue("Mobs", new String[]{"Mob"}, true);
     public final BooleanValue animals = new BooleanValue("Animals", new String[]{"Animal"}, true);
     public final BooleanValue vehicles = new BooleanValue("Vehicles", new String[]{"Vehic", "Vehicle"}, true);
+    public final BooleanValue endcrystals = new BooleanValue("EndCrystals", new String[]{"Crystals", "Cryst"}, true);
     public final BooleanValue local = new BooleanValue("Local", new String[]{"Self"}, true);
     public final BooleanValue items = new BooleanValue("Items", new String[]{"Item"}, true);
     public final BooleanValue footsteps = new BooleanValue("FootSteps", new String[]{"FootStep", "Steps"}, false);
     public final BooleanValue armorStand = new BooleanValue("ArmorStands", new String[]{"ArmorStand", "ArmourStand", "ArmourStands", "ArmStand"}, true);
 
-    public final BooleanValue name = new BooleanValue("Name", new String[]{"Nam"}, true);
-    public final BooleanValue ping = new BooleanValue("Ping", new String[]{"Ms"}, true);
-    public final BooleanValue armor = new BooleanValue("Armor", new String[]{"Arm"}, true);
-    public final BooleanValue hearts = new BooleanValue("Hearts", new String[]{"Hrts"}, true);
-    public final BooleanValue enchants = new BooleanValue("Enchants", new String[]{"Ench"}, true);
-    public final OptionalValue potions = new OptionalValue("Potions", new String[]{"Pot", "Pots", "PotsMode"}, 1, new String[]{"None", "Icons", "Text"});
-
-    public final BooleanValue background = new BooleanValue("Background", new String[]{"Bg"}, true);
-
     public final OptionalValue hpMode = new OptionalValue("Hp", new String[]{"Health", "HpMode"}, 0, new String[]{"None", "Bar", "BarText"});
 
     private ICamera camera = new Frustum();
-    private final ResourceLocation inventory = new ResourceLocation("textures/gui/container/inventory.png");
 
     //i cba
     private List<FootstepData> footstepDataList = new CopyOnWriteArrayList<>();
 
+    public static List<Entity> entitylist = new ArrayList<>();
```

Comment:
Making things static is just not a fix in any world.  Like I said, inconsistencies spill out everywhere when you require a constantly updated and fresh list between two modules that operate differently upon the list.  You could possibly try abstracting the rendering process to conveniently shrink all rendering iterations to one that calls upon different rendering methods.